code/iaas/service-discovery/server/src/main/java/io/cattle/platform/servicediscovery/process/ServiceActivate.java

```diff
@@ -68,15 +67,40 @@
     @Inject
     ServiceDiscoveryService sdServer;
 
+    @Inject
+    ServiceExposeMapDao svcExposeDao;
+
     @Override
     public String[] getProcessNames() {
-        return new String[] { ServiceDiscoveryConstants.PROCESS_SERVICE_ACTIVATE };
+        return new String[] { ServiceDiscoveryConstants.PROCESS_SERVICE_ACTIVATE,
+                ServiceDiscoveryConstants.PROCESS_SERVICE_UPDATE };
```

Comment:
I guess I'm not able to comment on a non-modified line in github (since I wanted to comment on the class name above).
Since we're adding 'update', I'm wondering whether we should modify the class name too to avoid confusion.  Super minor, so not merge blocking.